src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/loggers/PSFilterMetrics.java

```diff
@@ -0,0 +1,77 @@
+package org.broadinstitute.hellbender.tools.spark.pathseq.loggers;
+
+import htsjdk.samtools.metrics.MetricBase;
+
+import java.io.Serializable;
+
+/**
+ * Metrics that are calculated during the PathSeq filter
+ */
+public final class PSFilterMetrics extends MetricBase implements Serializable {
+
+    /**
+     * Number of input reads that are not secondary or supplementary alignments
+     */
+    public Long PRIMARY_READS;
```

Comment:
I wanted to use null's so I could tell if they have been assigned a value or not. I realized that since this class is public, it should check if they are null in computeDerivedMetrics() (before it was in the `PSScoreFileLogger` class). 

Similarly, PSScoreMetrics checks that the values are not null. If they are, however, the program will not crash. Rather, the metrics file writer will simply write an empty string for null values. 

I think is is the right behavior to output an empty string rather than writing, for example, 0, which is ambiguous.